src/main/kotlin/lotto/application/LottoMachine.kt

```diff
@@ -26,9 +26,11 @@ class LottoMachine(
         winningLotto: WinningLotto
     ): LottoResult {
         val winningResults = winningLotto.match(lottoBundle)
-        val winningAmount = winningResults.map { it.amount }
-            .reduce { acc, amount -> acc + amount }.value.div(purchase.amount.value.toDouble())
-        return LottoResult(winningAmount, winningResults.groupingBy { it }.eachCount())
+        val winningRate = winningResults.map { it.amount }
+            .reduce { acc, amount -> acc + amount }
+            .value
+            .div(purchase.amount.value.toDouble())
+        return LottoResult(winningRate, winningResults.groupingBy { it }.eachCount())
```

Comment:
```suggestion
        return LottoResult(
            winningRate = winningRate,
            winningResults = winningResults.groupingBy { it }
                .eachCount()
        )
```
파라미터가 어떤 것인지 잘 모르는 경우에는, argument name을 활용하면 좀 더 보기 편해요.